exist-installer/src/main/izpack/install.xml

```diff
@@ -172,7 +172,7 @@
         <pack name="shared" required="true" preselected="true" parent="Apps">
             <description>Shared Resources: jquery, dojo, ace, templating, utilities</description>
```

Comment:
Description should be updated as this is no longer correct

Reply:
done